### PR TITLE
Added: Nuxt website boilerplate

### PR DESCRIPTION
**Pull request by tonyspiro**
Thank you! 🙏

### CODE REVIEW

**Comment by tonyspiro** [comment]
Any update on this?

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
   <br>
 </p>
 
+YOO EDITING IN THE GitHub BROWSER
```
Comment (Geminii) [inline README.md:8]: Could be nice to remove this 🙈

Comment (Geminii) [reply README.md:8]: @tonyspiro Can you remove this 🤔

**Comment by Geminii** [comment]
Feel free to reopen a PR with taking consideration of the review 😇

**Pull request closed without merging**